code/cmdline/cmdline.cpp

```diff
@@ -1874,6 +1876,14 @@ bool SetCmdlineParams()
 		Cmdline_mod = modlist;
 	}
 
+	if(campaign_arg.found()) {
+		Cmdline_campaign = campaign_arg.str();
+
+		// strip off blank space if it's there
+		if ( Cmdline_campaign[strlen(Cmdline_campaign)-1] == ' ' ) {
```

Comment:
Will there be, at most, one space?  (Or tab, for that matter, if you want to use `is_gray_space()`.)

And, in fact, is this actually a problem in practice?  If so, we should double check that it's reasonable behavior for the command-line parser.

Reply:
After investigation, this bit of logic is copied from `-mod`, so it is known to work.  Whether it's the best approach is a question outside the scope of this PR.